### PR TITLE
Dave FIT-61

### PR DESCRIPTION
**Pull request by david-smith-osi**
Fix orphaned connection in IntakeLovCheck.

## Description
Health check not returning connections to the pool. Should run every 10 minutes, not every minute.

## Tests
- [x] I have included unit tests 
- [ ] I have included functional tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the google code style of this project.
- [x] I have ran all tests for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check SonarQube after the build completes, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: src/main/java/gov/ca/cwds/data/persistence/ns/utils/StringSequenceIdGenerator.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ public void configure(Type type, Properties params, ServiceRegistry serviceRegis
     }
 
     sequenceCallSyntax = dialect.getSequenceNextValString(sequenceName);
-    LOGGER.debug("sequence SQL: {}", sequenceCallSyntax);
+    LOGGER.info("sequence SQL: {}", sequenceCallSyntax);
```
Comment (igor-chornobay-engagepoint) [inline src/main/java/gov/ca/cwds/data/persistence/ns/utils/StringSequenceIdGenerator.java]: This probably should be a DEBUG

File: src/main/java/gov/ca/cwds/data/persistence/ns/utils/StringSequenceIdGenerator.java [hunk 2]
```diff
@@ -85,7 +85,7 @@ public Serializable generate(SharedSessionContractImplementor session, Object ob
         ((Number) Session.class.cast(session).createNativeQuery(sequenceCallSyntax).uniqueResult())
             .longValue();
 
-    LOGGER.debug("Next key: {}", seqValue);
+    LOGGER.info("Next key: {}", seqValue);
```
Comment (igor-chornobay-engagepoint) [inline src/main/java/gov/ca/cwds/data/persistence/ns/utils/StringSequenceIdGenerator.java]: This probably should be a DEBUG

File: src/main/java/gov/ca/cwds/data/persistence/xa/CandaceSessionFactoryImpl.java [hunk 3]
```diff
@@ -317,7 +317,7 @@ public Set getDefinedFilterNames() {
 
   @Override
   public void close() {
-    LOGGER.warn("******** CandaceSessionFactoryImpl.close ********");
+    LOGGER.info("******** CandaceSessionFactoryImpl.close ********");
```
Comment (igor-chornobay-engagepoint) [inline src/main/java/gov/ca/cwds/data/persistence/xa/CandaceSessionFactoryImpl.java]: This probably should be a DEBUG

File: src/main/java/gov/ca/cwds/data/persistence/xa/CandaceSessionImpl.java [hunk 4]
```diff
@@ -111,7 +124,7 @@ public Session getSession() {
 
   @Override
   public void close() throws HibernateException {
-    LOGGER.info("CandaceSessionImpl.close");
+    LOGGER.info("close");
```
Comment (igor-chornobay-engagepoint) [inline src/main/java/gov/ca/cwds/data/persistence/xa/CandaceSessionImpl.java]: This probably should be a DEBUG

File: src/main/java/gov/ca/cwds/data/persistence/xa/WorkFerbUserInfo.java [hunk 5]
```diff
@@ -68,15 +67,15 @@ public WorkFerbUserInfo(final boolean isDb2) {
 
   @Override
   public void execute(Connection con) throws SQLException {
-    LOGGER.warn("execute: Connection class: {}", con.getClass().getName());
+    LOGGER.info("execute: Connection class: {}", con.getClass().getName());
```
Comment (igor-chornobay-engagepoint) [inline src/main/java/gov/ca/cwds/data/persistence/xa/WorkFerbUserInfo.java]: not sure if it should be in logs all the time

File: src/main/java/gov/ca/cwds/data/persistence/xa/WorkFerbUserInfo.java [hunk 6]
```diff
@@ -68,15 +67,15 @@ public WorkFerbUserInfo(final boolean isDb2) {
...
     final RequestExecutionContext ctx = RequestExecutionContext.instance();
     final String staffId = ctx.getStaffId();
     final String userId = ctx.getUserId();
 
     if (isDb2 || con instanceof DB2Connection || (con instanceof PooledConnection
         && ((PooledConnection) con).getConnection() instanceof DB2Connection)) {
       try {
-        LOGGER.warn("DB2 connection, set user info: user id: {}, staff id: {}", userId, staffId);
+        LOGGER.info("DB2 connection, set user info: user id: {}, staff id: {}", userId, staffId);
```
Comment (igor-chornobay-engagepoint) [inline src/main/java/gov/ca/cwds/data/persistence/xa/WorkFerbUserInfo.java]: not sure if it should be in logs all the time

File: src/main/java/gov/ca/cwds/data/persistence/xa/WorkFerbUserInfo.java [hunk 7]
```diff
@@ -127,7 +127,8 @@ public void execute(Connection con) throws SQLException {
             final String resultWorkstation =
                 StringUtils.trimToEmpty(rs.getString("CUR_WORKSTATION"));
 
-            LOGGER.warn("client user: {}, application: {}, accounting: {}, workstation: {}",
+            LOGGER.info(
+                "client user: \"{}\", application: \"{}\", accounting: \"{}\", workstation: \"{}\"",
```
Comment (igor-chornobay-engagepoint) [inline src/main/java/gov/ca/cwds/data/persistence/xa/WorkFerbUserInfo.java]: not sure if it should be in logs all the time

File: src/main/java/gov/ca/cwds/data/persistence/ns/utils/StringSequenceIdGenerator.java (hunk 2 above)
Comment (david-smith-osi) [reply src/main/java/gov/ca/cwds/data/persistence/ns/utils/StringSequenceIdGenerator.java]: Agreed and done.

File: src/main/java/gov/ca/cwds/data/persistence/xa/WorkFerbUserInfo.java (hunk 7 above)
Comment (david-smith-osi) [reply src/main/java/gov/ca/cwds/data/persistence/xa/WorkFerbUserInfo.java]: We can downgrade the log level here soon, after gaining more experience with DB2 IDAA audit reports.

**Pull request merged**